Model/Action/ImportRatings.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace Lipscore\RatingsReviews\Model\Action;
+
+use Lipscore\RatingsReviews\Helper\Product;
+use Lipscore\RatingsReviews\Model\Api\RequestFactory;
+use Lipscore\RatingsReviews\Model\Config\AdminFactory;
+use Magento\Catalog\Model\ResourceModel\Product\Action as ProductActionResource;
+use Magento\Catalog\Model\ResourceModel\Product\Collection as ProductCollectionResource;
+use Magento\Catalog\Model\ResourceModel\Product\CollectionFactory;
+
+class ImportRatings
+{
+    const LIPSCORE_API_PRODUCT_ID_PARAM = 'internal_id';
+
+    protected $dataHelper;
+    protected $config;
+    protected $senderFactory;
+    protected $productActionResource;
+    protected $collectionFactory;
+    protected $api;
+
+    public function __construct(
+        ProductActionResource $productActionResource,
+        CollectionFactory $collectionFactory,
+        AdminFactory $adminConfigFactory,
+        RequestFactory $senderFactory
+    ) {
+        $this->config = $adminConfigFactory->create(
+            [
+                'storeId'   => 0,
+                'websiteId' => null
+            ]
+        );
+
+        $this->collectionFactory = $collectionFactory;
+        $this->productActionResource = $productActionResource;
+        $this->senderFactory = $senderFactory;
+    }
+
+    protected function getApi()
```

Comment:
we usually use the next approach on listing methods in classes:
- all the public methods first (an interface of a class)
- all the protected methods after public ones (internal functionality)